hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -53,8 +53,17 @@
 
     # Don't download default images, just our test images
     DOWNLOAD_DEFAULT_IMAGES=False
+    # Increase the total image size limit
+    GLANCE_LIMIT_IMAGE_SIZE_TOTAL=20000
     # We upload the Amphora image so it doesn't have to be build
-    IMAGE_URLS="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/amphora/2022-12-05/amphora-x64-haproxy.qcow2"
+    # Upload the images so we don't have to upload them from Prow
+    # NOTE: If you get issues when changing/adding images, check if the limits
+    # are sufficient and change the variable above if needed.
+    # https://docs.openstack.org/glance/latest/admin/quotas.html
+    IMAGE_URLS="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/amphora/2022-12-05/amphora-x64-haproxy.qcow2,"
+    IMAGE_URLS+="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/cirros/2022-12-05/cirros-0.6.1-x86_64-disk.img,"
+    IMAGE_URLS+="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/ubuntu/2023-09-29/ubuntu-2204-kube-v1.27.2.img,"
+    IMAGE_URLS+="https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/flatcar/flatcar-stable-3510.2.4-kube-v1.27.2.img"
```

Comment:
@mdbooth kindly[ uploaded a new version](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1713#issuecomment-1752768084), so we might want to probably use it here:

- `https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/flatcar/flatcar-stable-3602.2.0-kube-v1.27.2.img`

This would get us all the security fixes in that (and interim) release(s):

- https://www.flatcar.org/releases#release-3602.2.0

Reply:
Thanks for pointing this out. I have updated it :slightly_smiling_face: